src/tesk_core/filer.py

```diff
@@ -506,13 +627,16 @@ def main():
 
     for afile in data[args.transputtype]:
         logging.debug('Processing file: %s', afile['path'])
-        if process_file(args.transputtype, afile):
-            logging.error('Unable to process file, aborting')
-            return 1
+        process_file(args.transputtype, afile)
```

Comment:
Removing these lines will reintroduce a bug that was recently fixed.

Reply:
Reverted back